src/features/movies/moviesSlice.js

```diff
@@ -0,0 +1,50 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+const moviesSlice = createSlice({
+    name: "movies",
+    initialState: {
+        loading: true,
+        error: false,
+        movies: [],
```

Comment:
we need also initial state for single movie and reducer for it

Reply:
we have initialstate for movie here